Remotion/Development/UnitTesting.Compilation/Development.UnitTesting.Compilation.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectType>Library</ProjectType>
+  </PropertyGroup>
+  <Import Project="..\..\..\Build\Shared.build.props" />
+  <PropertyGroup>
+    <AssemblyTitle>re-motion Development Compilation Library for Unit Tests</AssemblyTitle>
+    <Description>Library for supporting compilation scenarios in Unit Tests.</Description>
+    <AssemblyName>Remotion.Development.UnitTesting.Compilation</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\Core\Core\Core.Core.csproj" />
```

Comment:
Is the only reason for the core-dependency the argument checks? if so, we could (should now or later @a-ctor?) replace it with a shared import. That way, the library becomes more isolated, easier to use in stand alone scenarios

Reply:
Yes only the argument checks are needed. I don't feel like it is necessary to remove the dependency on Core here. Chances are the consuming project already uses core if it uses the compilation library.

Reply:
okay, so leave it for now. eventually, I will try to get most of the Development stuff Core free so the Core tests can use development without first needing to compile Core. It's just a bit of cyclic dependency that's annoying but not harmful at this point.